docs/maps/index.asciidoc

```diff
@@ -12,16 +12,45 @@ With *Elastic Maps*, you can:
 * Create maps with multiple layers and indices.
 * Upload GeoJSON files into Elasticsearch.
 * Embed your map in Dashboards.
-* Plot individual documents or use aggregations to plot any data set, no matter how large.
 * Create choropleth maps.
-* Use data driven styling to symbolize features from property values.
+* Use data driven styling to symbolize features from data values.
 * Focus the data you want to display with searches.
 
 Start your tour of *Elastic Maps* with the <<maps-getting-started, getting started tutorial>>.
 
 [role="screenshot"]
 image::maps/images/sample_data_ecommerce.png[]
 
+[float]
+=== Create maps with multiple layers and indices
+Context is a force multiplier for spatial analysis, revealing how and where data sits relative to physical features like weather patterns, human-made features like international borders, and business-specific features like sales regions.
+Create maps with multiple layers and indices to show all of your data in a single map. Plot individual documents or use aggregations to plot any data set, no matter how large.
+
+[float]
+=== Upload GeoJSON files into Elasticsearch
+Elastic Maps makes it easy to import geospatial data into the Elastic Stack. Using the GeoJSON Upload feature, you can drag and drop your point and shape data files directly into Elasticsearch, and then use them as layers in the map.
+
+[float]
+=== Embed your map in Dashboards
+Viewing your data from different angles provides better insights. Dimensions that are obscured in one visualization may be illuminated from another. Add your maps to a dashboard and view your geospatial data alongside bar charts, pie charts, tag clouds, and more.
+
+[float]
+=== Create choropleth maps
+Plotting millions of individual documents on a map does not provide a lot of value. Documents overlap each other, making it difficult to extract insights.
+
+Choropleth maps are an ideal solution for visualizing big data sets. Choropleth maps provide a way to group documents into logical sets and visually compare and contrast metrics across groups allowing you to easily detect patterns. Choropleth maps shade areas in proportion to metric values and are commonly used to visualize population statistics such as race, age or political affiliation across geopolitical boundaries.
+
+[float]
+=== Use data driven styling to symbolize features from data values
+Customize each layer to highlight meaningful dimensions in your data. For example, use dark colors to symbolize areas with more web log traffic, and lighter colors to sybmolize areas with less traffic. Use larger circles to symbolize areas with more total bytes transferred, and smaller circles to symbolize areas with less bytes transferred.
```

Comment:
```suggestion
You can customize each layer to highlight meaningful dimensions in your data. For example, you can use dark colors to symbolize areas with more web log traffic, and lighter colors to symbolize areas with less traffic. .
```

Notes:

- Include the wording data-driven styling.  Otherwise, I don't see the connection.
- I didn't think the second example was necessary